pandas/src/parser/tokenizer.c

```diff
@@ -1876,3 +1886,88 @@ int64_t str_to_int64(const char *p_item, int64_t int_min, int64_t int_max,
     *error = 0;
     return number;
 }
+
```

Comment:
Again, _possible_ but not as nice because the implementations are definitely customized to the `dtype` (e.g. compare the difference in handling for "negative numbers" for `int64` and `uint64`).